turbopack/crates/turbopack-ecmascript-runtime/js/src/browser/runtime/base/dev-base.ts

```diff
@@ -212,6 +212,7 @@ function instantiateModule(moduleId: ModuleId, source: SourceInfo): Module {
           n: exportNamespace.bind(null, module, devModuleCache),
           m: module,
           c: devModuleCache,
+          C: null,
```

Comment:
I think we probably want to implement that for the browser runtime too, but that could be done later...